rpc/swap.go

```diff
@@ -74,7 +74,7 @@ type GetPastResponse struct {
 	ProvidedAmount *apd.Decimal        `json:"providedAmount"`
 	ExpectedAmount *apd.Decimal        `json:"expectedAmount"`
 	ExchangeRate   *coins.ExchangeRate `json:"exchangeRate"`
-	Status         string              `json:"status"`
+	Status         types.Status        `json:"status" validate:"required"`
```

Comment:
I didn't want to burden outstanding PRs with merge conflicts, so I only tagged the changed line with required.